build.gradle

```diff
@@ -302,37 +301,50 @@ allprojects {
       // compilationTask.source = compilationTask.source.sort()
 
       // Don't try to use `options.release.set(8)` here.
-      sourceCompatibility = 8
-      targetCompatibility = 8
-      // Because the target is 8, all of the public compiler classes are accessible, so
-      // --add-exports are not required (nor are they allowed with target 8). See
-      // https://openjdk.org/jeps/247 for details on compiling for older versions.
-
-      // When sourceCompatibility or release is changed to 11, then the following will be required.
-      // options.compilerArgs += [
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.model=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.processing=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED',
-      // '--add-exports', 'jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED',
-      // ]
-      // This is equivalent to writing "exports jdk.compiler/... to ALL-UNNAMED" in the
-      // module-info.java of jdk.compiler, so corresponding --add-opens are only required for
-      // reflective access to private members.
-      //
-      // From https://openjdk.org/jeps/261, Section titled: "Breaking encapsulation"
-      // "The effect of each instance [of --add-exports] is to add a qualified export of the
-      // named package from the source module to the target module. This is, essentially, a
-      // command-line form of an exports clause in a module declaration[...].
-      // [...]
-      // The --add-exports option enables access to the public types of a specified package.
-      // It is sometimes necessary to go further and enable access to all non-public elements
-      // via the setAccessible method of the core reflection API. The --add-opens option can
-      // be used, at run time, to do this."
+      // "true" is temporary, because omitting it causes wpiManyTest to fail, likely due to a bug in do-like-javac.
```

Comment:
Removing `true` would make the class files Java 11 and then the Checker Framework would no longer work with JDK 8.  It would be nice if the comment pointed that out.